beets/ui/commands.py

```diff
@@ -489,6 +494,13 @@ def choose_candidate(candidates, singleton, rec, cur_artist=None,
         assert cur_artist is not None
         assert cur_album is not None
 
+    # Build helper variables for extra choices.
+    # TODO: move "abort" choice always to the end?
+    # TODO: add plugin name info on choices ("(mb) Print tracks")?
+    extra_opts = tuple([c.long for c in extra_choices])
```

Comment:
FWIW, `tuple(c.long for c in extra_choices)` without the square brackets will work (using a "generator expression").